Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -57,7 +58,9 @@
 #include "ctkDICOMBrowser.h"
 #include "ctkDICOMObjectListWidget.h"
 #include "ctkDICOMQueryResultsTabWidget.h"
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.